helm/is-with-analytics/README.md

```diff
@@ -44,8 +42,10 @@ configurations as necessary.
     `sharedDeploymentLocationPath`: NFS shared deployment directory(<IS_HOME>/repository/deployment) location for IS<br> 
     `sharedTentsLocationPath`: NFS shared tenants directory(<IS_HOME>/repository/tenants) location for IS<br>
```

Comment:
@ThilinaManamgoda I think `sharedTentsLocationPath` should be renamed to `sharedTenantsLocationPath`. Since this is a value exposed to outside, it may confuse the users.

Reply:
will do